README.md

```diff
@@ -98,7 +98,7 @@ Simple usage of the module within your own main.tf file is as follows:
 | external\_project\_access\_violations\_should\_notify | Notify for External Project Access violations | string | `"true"` | no |
 | firewall\_rule\_enabled | Firewall rule scanner enabled. | string | `"true"` | no |
 | firewall\_rule\_violations\_should\_notify | Notify for Firewall rule violations | string | `"true"` | no |
-| folder\_id | GCP Folder that the Forseti project will be deployed into | string | `""` | no |
+| folder\_id | GCP Folder that the Forseti project will be deployed into | string | `""` | no** |
```

Comment:
This table is autogenerated, so we shouldn't make manual annotations.

I'd suggest we simply add a note at the TOP of the inputs table.

Reply:
You got it, will update

Reply:
I added a note. Let me know if we should change the wording.

Thanks